controllers/falcon_container/injector.go

```diff
@@ -0,0 +1,317 @@
+package falcon
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/crowdstrike/falcon-operator/apis/falcon/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crowdstrike/falcon-operator/pkg/common"
+	"github.com/crowdstrike/falcon-operator/pkg/tls"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	injectorName          = "injector"
+	initContainerName     = "crowdstrike-falcon-init-container"
+	injectorConfigMapName = "injector-config"
+	injectorTLSSecretName = "injector-tls"
```

Comment:
falconContainer.Name + injectorTLSSecretName

Reply:
This change is intended to maintain parity with the current implementation.
I'll look to update to more descriptive, and user manageable, resource names in the future.